doc/mgr/dashboard.rst

```diff
@@ -126,15 +126,26 @@ For example, a key pair can be generated with a command similar to::
 The ``dashboard.crt`` file should then be signed by a CA. Once that is done, you
 can enable it for all Ceph manager instances by running the following commands::
 
-  $ ceph config-key set mgr mgr/dashboard/crt -i dashboard.crt
-  $ ceph config-key set mgr mgr/dashboard/key -i dashboard.key
+  $ ceph config set mgr mgr/dashboard/crt -i dashboard.crt
+  $ ceph config set mgr mgr/dashboard/key -i dashboard.key
```

Comment:
No, these commands actually need to use `ceph config-key` (and I agree this is confusing)